omnibus.rb

```diff
@@ -39,6 +39,8 @@
 fetcher_retries 3
 fetcher_read_timeout 120
 
+workers 10
```

Comment:
Can we put a comment in here on why this is there? I'd rather have this not be a "magic number"